### DISCUSSION TITLE
Question on how to generate HL7 v3 CDA documents

### DISCUSSION THREAD

**Discussion by molinch** [FirelyTeam/firely-net-sdk#2875] [Q&A] [answered]
Dear maintainers,

The library is amazing and does a lot. I checked the history and found mentions of HL7 v3 CDA documents but it's not clear if you can directly generate such documents.

Could you tell me if the lib supports it ?
Thanks!

**Comment by mmsmits** [comment, marked as answer]
Hi @molinch,

Thanks for the compliments!
We have had some support for HL7 v3 CDA documents in the past, but we stopped actively supporting this a couple of years ago since no one was using it. I'm sorry.

**Reply by molinch** [reply]
Thanks for the feedback @mmsmits !
